test/common/network/dns_impl_test.cc

```diff
@@ -660,24 +661,6 @@ TEST_P(DnsImplTest, DnsIpAddressVersionV6) {
   dispatcher_->run(Event::Dispatcher::RunType::Block);
 }
 
-// Validate exception behavior during c-ares callbacks.
-TEST_P(DnsImplTest, CallbackException) {
-  // Force immediate resolution, which will trigger a c-ares exception unsafe
```

Comment:
any reason we can't keep the test but have it checking for the return status which we capture in the test class?

Reply:
Because these errors are made up... they aren't thrown from constructor of the Instance.

Reply:
not totally following; should we at least have some EXPECT_FALSE kind of checks for these inputs?

Reply:
This test inject some random callback that throw exception and verify the try catch behavior, but we expect the callback not to throw in the future.